File-Formats/DocIO/html.md

```diff
@@ -278,6 +278,103 @@ N> Calling the above event is mandatory in ASP.NET Core, UWP, and Xamarin platfo
+    using (WordDocument document = new WordDocument(docStream, FormatType.Docx))
+    {
+        //Hook the event to customize the image. 
+        document.SaveOptions.ImageNodeVisited += SaveImage;
+        using (FileStream outputStream = new FileStream("WordtoHTML.html", FileMode.Create, FileAccess.ReadWrite, FileShare.ReadWrite))
+        {
+            //Save the HTML file.
+            document.Save(outputStream, FormatType.Html);
+        }
+    }
+}
+
+{% endhighlight %}
+{% highlight c# tabtitle="C# [Windows-specific]" %}
+
+//Open an existing Word document. 
+using (WordDocument document = new WordDocument("Input.docx"))
+{
+    //Hook the event to customize the image. 
+    document.SaveOptions.ImageNodeVisited += SaveImage;
+    //Save a Word document as a HTML file.
+    document.Save("WordtoHTML.html", FormatType.Html);
+}
+
+{% endhighlight %}
+{% highlight vb.net tabtitle="VB.NET [Windows-specific]" %}
+
+'Open an existing Word document. 
+Using document As WordDocument = New WordDocument("Input.docx")
+    'Hook the event to customize the image. 
+    document.SaveOptions.ImageNodeVisited += SaveImage
+    'Save a Word document as a HTML file.
+    document.Save("WordtoHTML.html", FormatType.Html)
+End Using
+
+{% endhighlight %}
+{% endtabs %}
+
+The following code example illustrates the event handler for customizing the image path and saving the image in an external folder.
+
+{% tabs %}
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+
+static void SaveImage(object sender, ImageNodeVisitedEventArgs args)
+{
+    string imagepath = @"D:\Temp\Image.png";
+    //Save the image stream as a file.
+    using (FileStream fileStreamOutput = File.Create(imagepath))
+        args.ImageStream.CopyTo(fileStreamOutput);
+    //Set the URI to be used for the image in the output HTML. 
+    args.Uri = imagepath;
+}
+
+{% endhighlight %}
+{% highlight c# tabtitle="C# [Windows-specific]" %}
+
+static void SaveImage(object sender, ImageNodeVisitedEventArgs args)
+{
+    string imagepath = @"D:\Temp\Image.png";
+    //Save the image stream as a file. 
+    using (FileStream fileStreamOutput = File.Create(imagepath))
+        args.ImageStream.CopyTo(fileStreamOutput);
+    //Set the image URI to be used in the output HTML.
+    args.Uri = imagepath;
+}
+
+{% endhighlight %}
+{% highlight vb.net tabtitle="VB.NET [Windows-specific]" %}
+
+Private Shared Sub SaveImage(ByVal sender As Object, ByVal args As ImageNodeVisitedEventArgs)
+    Dim imagepath = "D:\Temp\Image.png"
+    'Save the image stream as a file. 
+    Using fileStreamOutput = File.Create(imagepath)
+        args.ImageStream.CopyTo(fileStreamOutput)
+    End Using
+    'Set the URI to be used for the image in the output HTML. 
+    args.Uri = imagepath
+End Sub
+
+{% endhighlight %}
+{% endtabs %}
+
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/HTML-conversions/Customize-image-path-in-Word-to-html/.NET).
+
```

Comment:
Tips:
By utilizing the event handler mentioned above, you can also implement logic to store images in the Cloud or other online storage platforms.

Reply:
Added